PlayerAdministration.cs

```diff
@@ -1619,38 +1727,60 @@ private void BuildUserPage(ref Cui aUIObj, UiPage aPageType, ulong aPlayerId)
                     );
                 }
 
-                if (VerifyPermission(aUIObj.PlayerIdString, CPermSpectate)) {
+                if (VerifyPermission(aUIObj.PlayerIdString, CPermSpectate))
+                {
                     aUIObj.AddButton(
                         actionPanel, CUserPageBtnSpectateLbAnchor, CUserPageBtnSpectateRtAnchor, CuiColor.ButtonSuccess, CuiColor.TextAlt,
                         lang.GetMessage("Spectate Player Button Text", this, aUIObj.PlayerIdString), $"{CSpectateUserCmd} {aPlayerId}"
                     );
-                } else {
+                }
+                else
+                {
                     aUIObj.AddButton(
                         actionPanel, CUserPageBtnSpectateLbAnchor, CUserPageBtnSpectateRtAnchor, CuiColor.ButtonInactive, CuiColor.Text,
                         lang.GetMessage("Spectate Player Button Text", this, aUIObj.PlayerIdString)
                     );
                 }
 
                 // Perms
-                if (PermissionsManager == null) {
+                if (PermissionsManager == null)
+                {
                     aUIObj.AddButton(
                         actionPanel, CUserPageBtnPermsLbAnchor, CUserPageBtnPermsRtAnchor, CuiColor.ButtonInactive, CuiColor.Text,
                         lang.GetMessage("Perms Not Installed Button Text", this, aUIObj.PlayerIdString)
                     );
-                } else if (VerifyPermission(aUIObj.PlayerIdString, CPermPerms)) {
+                }
+                else if (VerifyPermission(aUIObj.PlayerIdString, CPermPerms))
+                {
                     aUIObj.AddButton(
                         actionPanel, CUserPageBtnPermsLbAnchor, CUserPageBtnPermsRtAnchor, CuiColor.ButtonSuccess, CuiColor.TextAlt,
                         lang.GetMessage("Perms Button Text", this, aUIObj.PlayerIdString), $"{CPermsCmd} {aPlayerId}"
                     );
-                } else {
+                }
+                else
+                {
                     aUIObj.AddButton(
                         actionPanel, CUserPageBtnPermsLbAnchor, CUserPageBtnPermsRtAnchor, CuiColor.ButtonInactive, CuiColor.Text,
                         lang.GetMessage("Perms Button Text", this, aUIObj.PlayerIdString)
                     );
                 }
 
+                // Backpacks & Inventory
```

Comment:
It'd be nice if you used an `if {} else {}` here, like the rest of the code (Compiler will now make 3 if clauses). Looks fine otherwise.

Reply:
Alright coolstuf, will change this to be inline with the rest of the code.